data/mods/MindOverMatter/help_files.json

```diff
@@ -35,7 +35,7 @@
     "name": "Mind Over Matter: Gaining Psionic Powers",
     "messages": [
       "There are three main ways to gain psionic powers: start the game with them, find and use a matrix crystal, or exposure yourself to the raging energies of a portal storm.",
-      "The first and most obvious is start the game with them.  There are professions for each psionic path, such as \"Doomseer\" for clairsentience or \"Itzcuauhtli Corps Liaison, Yohualli Ehēcatl Division\" for teleportation, and each path also has a series of three backgrounds entitled Newly-Awakened, Arising, and Ascended.  Newly-Awakened starts you with the absolute minimum possible, only a couple powers.  Arising starts you with a good amount of powers, equivalent to a mid-game character.  Ascended starts you with a very powerful suite of psionics best suited for games where monster difficulty has been significantly increased or other challenges are at play.",
+      "The first and most obvious is start the game with them.  There are professions for each psionic path, such as \"Doomseer\" for clairsentience or \"Itzcuāuhtli Corps Liaison, Yohualli Ehēcatl Division\" for teleportation, and each path also has a series of three backgrounds entitled Newly-Awakened, Arising, and Ascended.  Newly-Awakened starts you with the absolute minimum possible, only a couple powers.  Arising starts you with a good amount of powers, equivalent to a mid-game character.  Ascended starts you with a very powerful suite of psionics best suited for games where monster difficulty has been significantly increased or other challenges are at play.",
```

Comment:
Does the ā render in all our provided fonts? Otherwise these accents might've been excluded for that reason

Reply:
If not then the place already using the ā should be changed to match here?